dwave/system/utilities.py

```diff
@@ -17,9 +17,13 @@
 import os
 import json
 import networkx as nx
+import numpy as np
 import warnings
 
-__all__ = ['common_working_graph', 'classproperty']
+__all__ = [
+    'common_working_graph',
+    'classproperty',
+    'schedule_with_anneal_offset']
```

Comment:
Either
```suggestion
    'schedule_with_anneal_offset',
    ]
```
or
```suggestion
    'schedule_with_anneal_offset',
]
```
See [pep8](https://peps.python.org/pep-0008/#indentation).